common/src/api/external/mod.rs

```diff
@@ -808,6 +799,17 @@ impl<'de> Deserialize<'de> for Generation {
     }
 }
 
+// This is the equivalent of applying `#[serde(transparent)]`, but that has a
+// side effect of changing the JsonSchema derive to no longer emit a schema.
+impl Serialize for Generation {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        self.0.serialize(serializer)
+    }
+}
```

Comment:
I want to call out this change -- I believe there is a bug in progenitor 0.9.x where newtype structs that do not have `#[serde(transparent)]` cannot be serialized in a query string, and I need to go file an issue for it. But I think in practice it is more accurate to manually implement `Serialize` in Omicron.